.github/workflows/nodejs.yml

```diff
@@ -6,7 +6,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        node-version: [14.x, 15.x]
+        node-version: [14.x, 16.x]
```

Comment:
While here, I'd give `actions/setup-node@v2` a try.  Supposedly it's faster.  Could even test it with `cache: 'yarn'` under `with:` to see if that speeds anything up.